src/Sulu/Bundle/ContentBundle/Resources/js/views/PageEdit/PageEdit.js

```diff
@@ -0,0 +1,166 @@
+// @flow
+import React from 'react';
+import {action, observable} from 'mobx';
+import {observer} from 'mobx-react';
+import {FormStore, Form, withToolbar} from 'sulu-admin-bundle/containers';
+import type {ViewProps} from 'sulu-admin-bundle/containers';
+import {translate} from 'sulu-admin-bundle/utils';
+import {ResourceStore} from 'sulu-admin-bundle/stores';
+import WebspaceStore from '../../stores/WebspaceStore';
+import type {Webspace} from '../../stores/WebspaceStore/types';
+import pageEditStyles from './pageEdit.scss';
+
+type Props = ViewProps & {
+    resourceStore: ResourceStore,
+};
+
+@observer
+class PageEdit extends React.Component<Props> {
```

Comment:
`PageForm` (and don't forget to rename the file, caused me some troubles in the past :confused:)

Reply:
And I think this component misses tests